SU-SWS/decanter
# D8CORE-1403: addressing accessibility issue with footer

# READY FOR REVIEW


# Summary
This addresses HTML errors from footer `<section>` by changing them to `<div>`
This adds a screenreader only  `<h2>Address</h2>` to make it easier to find the address using headings.

# Needed By (Date)


# Urgency
Nice to have, addresses an accessibility issue. 

# Steps to Test


1. Pull this branch
1. Use https://validator.w3.org/ to verify that 
```
Warning: Section lacks heading. Consider using h2-h6 elements to add identifying headings to all sections.
From line 633, column 1; to line 633, column 35
 </div>↩↩↩<section class="su-global-footer ">↩  <di
```
no longer appears
1. Verify that "Address" does not appear on the rendered page, but is available in the HTML.


# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1403
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
